homeassistant/components/light/plum_lightpad.py

```diff
@@ -0,0 +1,251 @@
+        self._green = lightpad.glow_color['green']
+        self._blue = lightpad.glow_color['blue']
+        self._white = lightpad.glow_color['white']
+        self._brightness = lightpad.glow_intensity * 255.0
+        lightpad.add_event_listener('configchange', self.configchange_event)
+
+    def configchange_event(self, event):
+        """Configuration change event handling."""
+        config = event['changes']
+        self._red = config['glowColor']['red']
+        self._green = config['glowColor']['green']
+        self._blue = config['glowColor']['blue']
+        self._white = config['glowColor']['white']
+        self._brightness = config['glowIntensity'] * 255.0
+        self.schedule_update_ha_state()
+
+    @property
+    def red(self):
+        """Return the Red value 0..255."""
+        return self._red
+
+    @property
+    def green(self):
+        """Return the Green value 0..255."""
+        return self._green
+
+    @property
+    def blue(self):
+        """Return the Blue value 0..255."""
+        return self._blue
+
+    @property
+    def white_value(self):
+        """White Color value from 0..255."""
+        return self._white
+
+    @property
+    def rgb_color(self):
+        """RGB Property."""
+        return self.red, self.green, self.blue
+
+    @property
+    def hs_color(self):
+        """Return the hue and saturation color value [float, float]."""
+        return color_util.color_RGB_to_hs(self.red, self.green, self.blue)
+
+    @property
+    def should_poll(self):
+        """No polling needed."""
+        return False
+
+    @property
+    def name(self):
+        """Return the name of the switch if any."""
+        return self._name
+
+    @property
+    def brightness(self) -> int:
+        """Return the brightness of this switch between 0..255."""
+        return self._brightness
+
+    @property
+    def glow_intensity(self):
+        """Brightness in float form."""
+        return self._brightness / 255.0
+
+    @property
+    def is_on(self) -> bool:
+        """Return true if light is on."""
+        return self._lightpad.glow_enabled
+
+    @property
+    def icon(self):
+        """The crop-portait icon works like the glow ring."""
+        return 'mdi:crop-portrait'
+
+    @property
+    def supported_features(self):
+        """Flag supported features."""
+        return SUPPORT_BRIGHTNESS | SUPPORT_COLOR
+
+    @property
+    def device_state_attributes(self):
+        """Additional attributes listed for the device."""
+        return {
+            'red': self._red,
+            'green': self._green,
+            'blue': self._blue,
+            'white': self._white,
+            'glowTimeout': self._lightpad.glow_timeout,
+            'glowFade': self._lightpad.glow_fade,
+        }
+
+    def turn_on(self, **kwargs):
+        """Turn the light on."""
+        if ATTR_BRIGHTNESS in kwargs:
+            self._brightness = kwargs[ATTR_BRIGHTNESS]
```

Comment:
See comment above in the other entity class.

Reply:
Good call, I missed this one.